src/Drush/Commands/Commands.php

```diff
@@ -9,32 +9,27 @@
 use Drush\Commands\DrushCommands;
 use Drush\Exceptions\CommandFailedException;
 use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drush\Commands\AutowireTrait;
+use Symfony\Component\DependencyInjection\Attribute\Autowire;
 
 /**
  * Drush commands.
  */
 final class Commands extends DrushCommands {
 
+  use AutowireTrait;
+
   /**
    * Commands constructor.
    */
   public function __construct(
+    #[Autowire(service: \Drupal\azure_ad_delta_sync\Controller::class)]
```

Comment:
Use `use` to use `Drupal\azure_ad_delta_sync\Controller` and rewrite:

```suggestion
    #[Autowire(service: Controller::class)]
```